public/static/js/spawned_hosts.js

```diff
@@ -221,6 +224,28 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       );
     };
 
+    $scope.addNewVolumes = function () {
+        mciSpawnRestService.getVolumes(
+            {
+                success: function (resp) {
+                    var volumes = resp.data;
+                    _.each(volumes, function (volume) {
+                        var foundVolume = false;
+                        for (var i = 0; i < $scope.volumes.length; i++) {
+                            if ($scope.volumes[i].id === volume.id) {
+                                foundVolume = true;
+                                break;
+                            }
+                        }
+                        if (!foundVolume) {
+                            $scope.volumes.push(volume);
```

Comment:
do we need to do the processing we do [here](https://github.com/evergreen-ci/evergreen/blob/25c2a3b9124b91bd224455bb7564993b0e8f0696/public/static/js/spawned_hosts.js#L194) for the new volume?
(e.g. setting up the display name fields)